client/src/components/molecules/NetworkSwitcher.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useContext, useEffect, useRef } from 'react'
+import { urlq } from '../../utils/utils'
+import { CONNECTIONS } from '../../config'
+import { User } from '../../context'
+import styles from './NetworkSwitcher.module.scss'
+
+const networkUrlParam = urlq.get('network') || ''
+
+const getNetworkConfig = (network: string) => {
+    const index = Object.keys(CONNECTIONS).indexOf(network)
+    // TypeScript doesn't let access CONNECTIONS[networkName] directly
+    return index !== -1
+        ? Object.values(CONNECTIONS)[index]
+        : CONNECTIONS.pacific // Use default config in case of mispelled URL params or
+}
+
+export const oceanConfig =
+    networkUrlParam !== ''
+        ? getNetworkConfig(networkUrlParam)
+        : getNetworkConfig('pacific')
+
+/* NETWORK SWITCHER */
+export function NetworkSwitcher() {
+    const node: any = useRef()
+    const [isToggled, setIsToggled] = useState(false)
+
+    useEffect(() => {
+        // Handle click outside to collapse Network switcher dropdown
+        // add listener when mounted
+        document.addEventListener('mousedown', handleTogle)
+        // return function when unmounted
+        return () => {
+            document.removeEventListener('mousedown', handleTogle)
+        }
+    }, [])
+    /*  
+    useEffect(() => {
+        if (networkUrlParam !== '') {
+            switchNetwork(networkUrlParam)
+        }
+    }, []) 
+    */
+
+    const handleTogle = (e: any) => {
+        const isClickedInside = node.current.contains(e.target)
+        setIsToggled(isClickedInside)
+    }
+
+    const { network, isBurner } = useContext(User)
+
+    console.log(isBurner)
+
+    const switchNetwork = (networkName: string): any => {
+        // Force page to get refreshed
+        window.location.href = `${window.location.origin}?network=${networkName}`
+        //userContext.switchNetwork(networkName, getNetworkConfig(networkName))
+        setIsToggled(false) // for the case without force page refresh
+    }
+
+    return !isBurner ? null : (
+        <div
+            ref={node}
+            className={`${styles.networkListWrapper} ${
+                isToggled ? styles.on : ''
+            }`}
+        >
+            <em onClick={e => handleTogle(e)}>
+                <span>Change Network</span>
+            </em>
+            <ul className={styles.networkList}>
+                {Object.keys(CONNECTIONS).map((networkName, i) => (
+                    <li
+                        key={i}
```

Comment:
just super minor but there can be [some issues with using `index` as unique `key`](https://menubar.io/react-keys-index/). To work around that, whenever we need a unique key we have a package [`shortid`](https://www.npmjs.com/package/shortid). So instead of `i` here it can use that package's `shortid.generate()`

Reply:
I simply changed it to the network name :)